### PR TITLE
document sequence operator variants ^... and ^...^ ref #3360

### PR DESCRIPTION
**Pull request by stoned**
while here add output to an example for this operator

### CODE REVIEW

File: doc/Language/operators.pod6 [hunk 1]
```diff
@@ -3148,11 +3155,12 @@ knows about arithmetic and geometric sequences.
 If the endpoint is not C<*>, it's smartmatched against each generated
 element and the sequence is terminated when the smartmatch succeeded.
 For the C<...> operator, the final element is included, for the C<...^>
-operator it's excluded.
+and the C<^...^> operators it's excluded.
```
Comment (JJ) [inline doc/Language/operators.pod6:3158]: I guess that, for symmetry, you should also mention the ^… operator, right?

**Review by JJ — APPROVED** [review]
It's OK, please address my comment before merging if you consider it necessary.

File: doc/Language/operators.pod6 (hunk 1 above)
Comment (stoned) [reply doc/Language/operators.pod6]: Hum... I think I see what you mean. What do you think about the commit f8fd3cc I pushed on this branch ?

Comment (JJ) [reply doc/Language/operators.pod6]: Essentially yes, but check out again my comments, sorry :-)

Comment (stoned) [reply doc/Language/operators.pod6]: no worries. :-)

**Pull request merged**